.eslintrc.ci.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": ".eslintrc.json",
+  "rules": {
+    "linebreak-style": 2
+  }
+}
```

Comment:
Not sure if the line ending should be added

Reply:
That file was copied from the wet-boew/wet-boew repo, so it should probably be addressed there. I'll likely be updating this PR soon to remove that file.